app/models/project.rb

```diff
@@ -84,9 +84,14 @@ def establishment
   end
 
   def incoming_trust
+    return new_trust_object if form_a_mat?
     @incoming_trust ||= fetch_trust(incoming_trust_ukprn)
   end
 
+  def new_trust_object
```

Comment:
Sneaky! I think I like this!

Reply:
Haha seemed like the easiest thing to do to ensure it all works as expected!